test/unit/calculators/statutory_sick_pay_calculator_test.rb

```diff
@@ -1048,11 +1049,53 @@ class StatutorySickPayCalculatorTest < ActiveSupport::TestCase
           end
         end
 
+        context "starting and ending before 13 March 2020" do
+          start_date = Date.parse("9 March 2020") # thursday
+          end_date = Date.parse("12 March 2020") # tuesday
+
+          should "use normal rules for SSP if they're ill before entitlement date" do
+            calc = StatutorySickPayCalculator.new(
+              sick_start_date: start_date,
+              sick_end_date: end_date,
+              days_of_the_week_worked: %w(1 2 3 4 5),
+              has_linked_sickness: false,
+              coronavirus_related: true,
+              has_coronavirus: true,
+            )
+
+            assert_equal 1, calc.days_paid
+            assert_equal false, calc.ends_after_coronavirus_entitlement_date?
```

Comment:
I'd suggest this is more relevant - or we could have both:

```suggestion
            assert_equal true, calc.before_coronavirus_shield_date?
```

Reply:
We can have both.